pkg/collector/corechecks/system/cpu/cpu_windows.go

```diff
@@ -149,12 +148,12 @@ func (c *Check) Configure(integrationConfigDigest uint64, data integration.Data,
 	}
 
 	// do nothing
-	info, err := cpuInfo()
+	info := cpuInfo()
+	cpucount, err := info.CPULogicalProcessors.Value()
 	if err != nil {
-		return fmt.Errorf("cpu.Check: could not query CPU info")
+		return fmt.Errorf("cpu.Check: could not get number of CPU")
 	}
-	cpucount, _ := strconv.ParseFloat(info["cpu_logical_processors"], 64)
-	c.nbCPU = cpucount
+	c.nbCPU = float64(cpucount)
```

Comment:
I wondered too, my best guess is that it was used to compute the number of cpu cycles, and it avoided converting it to a float when doing the division
It's been like that since the creation of the file in https://github.com/DataDog/datadog-agent/pull/1725